src/index.ts

```diff
@@ -88,8 +92,8 @@ export type Context = {
   request?: object;
   info?: object;
   error?: object;
-  prev?: object;
-  stash?: object;
+  prev?: object & { result: object };
```

Comment:
```suggestion
  prev?: object;
```

I don't think it makes sense to include the result key, as it won't always be defined.